lib/challenge.js

```diff
@@ -19,6 +19,36 @@
   // - park
 
 // Answer Starts Here
+/ Description: Our app keeps track of things on a calender
+// Every 'Calender' has
+// - months
+// - weeks
+//  - days
+// - dates
+// - events
+// Every calender can
+// - display schedule
+// - add new events
+// - modify events
```

Comment:
Well done. The question asked for three or more, so this counts for half credit.